contracts/token/ERC721/ERC721Pausable.sol

```diff
@@ -8,6 +8,8 @@ import "../../utils/Pausable.sol";
  * @dev ERC721 modified with pausable transfers.
  */
 contract ERC721Pausable is ERC721, Pausable {
+    constructor (string memory name, string memory symbol) public ERC721(name, symbol) { }
```

Comment:
This is not good. I don't think it will be possible to combine both `ERC721Pausable` and `ERC721Burnable` because they both initialize `ERC721`. Can you check?

This is the `Crowdsale` situation all over again. The "solution" is to not initialize `ERC721` in here, rather leave it abstract. As a result, `ERC721Pausable` will not be usable out of the box, and if someone wants to use it they will have to combine it themselves with an instantiation of `ERC721`. I don't think this is that bad if we alternatively ship the batteries included contracts. By doing that we would be saying "we know that the other contracts are not usable out of the box, here's some that are".

If we go for this, it's possible that `ERC721Burnable` would not make much sense due to potential `override` issues (it's not clear to me if they will show up in this case).

Reply:
What would I do without you! Yes, you're absolutely right. I gave it a go on Remix and this is the only way I was able to get it to work: 

```solidity
pragma solidity ^0.6.0;

contract Base {
    uint256 private _a;
    
    constructor(uint256 a) public {
        _a = a;
    }
}

abstract contract A is Base {
}

abstract contract B is Base {
}

contract C is A, B {
    constructor(uint256 a) public Base(a) { }
}
```

I agree that making both `abstract` is the best way to address this. I'm not sure though why you say this will cause issues with `Burnable`, could you expand on this?

Reply:
I thought about the Burnable thing a bit more and I think I was mistaken. I thought there would be a conflict but Burnable adds a new function so it doesn't conflict with anything.